### PR TITLE
chore: add default oathkeeper accessRules

### PR DESCRIPTION
**Pull request by openoms**
Partial changes from the reverted PR: https://github.com/GaloyMoney/charts/pull/2474

Changing only the Oathkeeper accessRules.

These settings are not overwritten in deployments.

This commit shows difference between the old (expanded from the one line) and new AccessRules: https://github.com/GaloyMoney/charts/pull/2515/commits/dd0d61428bd81fe958cca0e8769ae3df7f573623

### CODE REVIEW

**Review by krtk6160 — APPROVED** [review]

**Comment by openoms** [comment]
@ntheile since the in the galoy values:
the 
kratos:
  kratos:
      serve:
        public:
          base_url: http://127.0.0.1:4433/
        admin:
          base_url: http://0.0.0.0:4434/
```
The access oathkeeper accessRules would also need to point to  4433 instead of 80.
Alternatively we need to change to 80 everywhere, but the accessRules would still use a different port on staging:

Looking in deployments the `kratos_public_host` is actually exposed via ingress on `https://` which implies the port 443.

**Comment by openoms** [comment]
@ntheile do we need to change the

https://github.com/GaloyMoney/charts/pull/2515/files#diff-bb75812aa7e2bf04250760613379fce958cb74bafba89126e4fde31e78b3376eL599

File: charts/galoy/values.yaml [hunk 1]
```diff
@@ -798,7 +802,7 @@ oathkeeper:
             {
               "handler": "cookie_session",
               "config": {
-                "check_session_url": "http://galoy-kratos-public:80/sessions/whoami",
```
[the remark this answers is not in the document]
Comment (openoms) [reply charts/galoy/values.yaml]: that's right. I have tested both locally and on staging. The service is called `galoy-kratos-public` and serving on the port 80. 
This has been probably a misconfiguration on staging which we did not catch due it wasn't being used.

[the remark this answers is not in the document]
Comment (openoms) [reply charts/galoy/values.yaml]: changed back in [acc33e5](https://github.com/GaloyMoney/charts/pull/2515/commits/acc33e579840970162b06c4dbb0ee3e5197446a1)

**Comment by openoms** [comment]
Now the only the galoy charts values.yaml is the single source of truth for the oathkeeper rules.

**Pull request merged**